course_catalog/migrations/0003_auto_20190108_2355.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.0.8 on 2019-01-08 23:55
```

Comment:
Sure, we can rename them. Though, since there's no living production databases yet, we have the option of blowing away (resetting) the migrations, if you like.

Reply:
That option is fine with me too, feel free to pick whichever is easier.